### DISCUSSION TITLE
problem with creation of 2nd plot stucked at 100%

### DISCUSSION THREAD

**Discussion by S1m0n3tr3** [Chia-Network/chia-blockchain#4506] [Support]
Hello,I have a problem with the creation of the second plot

![chia plot2bug2](https://user-images.githubusercontent.com/36471664/117738993-9cc60f00-b1fd-11eb-8630-7a008466d2b4.png)
![chia plot2bug](https://user-images.githubusercontent.com/36471664/117739002-9f286900-b1fd-11eb-8067-e3773af1e83a.png)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
